src/RequestForm.js

```diff
@@ -247,25 +243,13 @@ class RequestForm extends React.Component {
   }
 
   onChangeFulfilment(e) {
-    const {
-      defaultDeliveryAddressTypeId,
-      defaultServicePointId,
-    } = this.state;
-    const { change } = this.props;
-
-    const deliverySelected = e.target.value === fulfilmentTypeMap.DELIVERY;
+    const selectedFullfillmentPreference = e.target.value;
 
     this.setState({
-      selectedDelivery: deliverySelected,
+      selectedDelivery: this.isDeliverySelected(selectedFullfillmentPreference),
```

Comment:
Please, rename `selectedDelivery` everywhere in file to §deliveryIsSelected` or `deliverySelected`.